src/coreclr/vm/ceeload.cpp

```diff
@@ -4778,6 +4810,30 @@ VASigCookie *Module::GetVASigCookie(Signature vaSignature)
             pCookie->pNDirectILStub = NULL;
             pCookie->sizeOfArgs = sizeOfArgs;
             pCookie->signature = vaSignature;
```

Comment:
There are more places that need to be switched to use loader module instead of module to allocate memory associated with the VASigCookie.

For example, this place: https://github.com/dotnet/runtime/blob/443ece48d04eff6f09c9ab34e45fbb20b457f4e8/src/coreclr/vm/dllimport.cpp#L6023